doc/source/user_guide/session/launching_ansys_fluent.rst

```diff
@@ -380,3 +380,155 @@ Connecting to a Fluent container running inside Linux from a Windows host
 
   >>> import ansys.fluent.core as pyfluent
   >>> solver = pyfluent.connect_to_fluent(ip="10.18.19.151", port=44383, password="hbsosnni")
+
+
+Connecting to Fluent on Windows from a Linux or WSL host
+--------------------------------------------------------
+
+This guide describes how to connect to an ANSYS Fluent instance running on a Windows machine from a Linux or WSL host. 
+It also includes steps to enable remote file transfer.
+
+  Prerequisites:
+
+    - `Docker <https://www.docker.com/>`_
+    - `Build file transfer server <https://filetransfer-server.tools.docs.pyansys.com/version/stable/intro.html#>`_
+
+A. **Set Up Fluent and File Transfer Server on Windows**
+
+1. **Launch Fluent**
+
+   Open a command prompt and run:
+
+   .. code:: console
+
+      ANSYS Inc\v252\fluent\ntbin\win64\fluent.exe 3ddp -sifile=server_info.txt
+      type server_info.txt
+
+   Example output:
+   ``10.18.44.179:51344``, ``5scj6c8l``
+
+2. **Retrieve Connection Details**
+
+   Get the IP address, port, and password from the `server_info.txt` file.  
+   Example:
+   - IP: ``10.18.44.179``
+   - Port: ``51344``
+   - Password: ``5scj6c8l``
+
+3. **Set Working Directory**
+
+   Change to the Fluent installation directory:
```

Comment:
Is changing to Fluent installation directory necessary? User may not have write permission at the installation directory.

Fluent's working directory can be anywhere. The only requirement is that the docker container should be run from Fluent's working directory.

Reply:
Done. Thank you.